### DISCUSSION TITLE
Footer

### DISCUSSION THREAD

**Discussion by stephanlamoureux** [Vets-Who-Code/march2021#56] [Ideas]
Should we add everyone's GitHub profile to the footer?

**Comment by dtetreau251** [comment]
Absolutely 


Sent from Yahoo Mail for iPhone


On Wednesday, March 10, 2021, 8:18 PM, Stephan Lamoureux ***@***.***> wrote:




Should we add everyone's GitHub profile to the footer?

**Comment by joeromine** [comment]
I don't like the border on the bottom of the footer.

**Comment by dtetreau251** [comment]
this was handled by steve

**Comment by joeromine** [comment]
Thank You
